avocado/utils/linux_modules.py

```diff
@@ -39,17 +39,18 @@
 BUILTIN = 2
 
 
-def load_module(module_name):
+def load_module(module_name, parameters=''):
     """
     Checks if a module has already been loaded.
     :param module_name: Name of module to check
+    :param parameters: Additional parameters to load module with
     :return: True if module is loaded, False otherwise
     :rtype: Bool
     """
     if module_is_loaded(module_name):
         return False
 
-    process.system('/sbin/modprobe ' + module_name)
+    process.system('/sbin/modprobe %s %s' % (module_name, parameters))
```

Comment:
@ldoktor The `/sys/class///parameter` is not exactly a symlink to `/sys/module//parameters/parameter`
```
# cat /sys/class/scsi_host/host1/lpfc_link_speed
0
# cat /sys/module/lpfc/parameters/lpfc_link_speed
0

# rmmod lpfc; modprobe lpfc lpfc_link_speed=1

# cat /sys/module/lpfc/parameters/lpfc_link_speed
1
# cat /sys/class/scsi_host/host1/lpfc_link_speed
0
```